packages/resizin/src/buildUrl/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from './buildUrl';
+export { default as buildUrlFactory } from './factory';
```

Comment:
The build fails on this line - but it is fixed already. Be sure build wont fail

Reply:
Yeah I know, there were (and maybe still is) some trouble with build setup